frontend/src/tests/components/noticeOfWork/__snapshots__/NOWActivities.spec.js.snap

```diff
@@ -108,25 +108,18 @@ exports[`NOWActivities renders properly 1`] = `
               <p
                 className="field-title"
               >
-                Are you proposing any bridges, culverts, and crossings?
+                Describe the proposed reclamation and timing for this specific activity
               </p>
             </Col>
             <Col
               md={12}
               xs={24}
             >
               <p>
-                Unknown
+                N/A
```

Comment:
Did we not say that we were going to use dashes?

Reply:
chatted IRL - let's consult the designers